src/org/labkey/test/BootstrapLocators.java

```diff
@@ -0,0 +1,9 @@
+package org.labkey.test;
+
+public abstract class BootstrapLocators
+{
+    static public Locator infoAlertLoc = Locator.tagWithClass("div", "alert-info");
+    static public Locator successAlertLoc = Locator.tagWithClass("div", "alert-success");
+    static public Locator dangerAlertLoc = Locator.tagWithClass("div", "alert-danger");
+    static public Locator warningAlertLoc = Locator.tagWithClass("div", "alert-warning");
```

Comment:
I'd say remove the `"Loc"` suffix from these; its kinda redundant with the class name.

Reply:
done